src/plugins/__tests__/__snapshots__/react-jss.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`1. basic 1`] = `
+"
+import React from 'react'
+import injectSheet from 'react-jss'
+
+const styles = {
+  button: {
+    fontWeight: 'bold',
+  },
+}
+
+const Button = ({classes, children}) =>
+  React.createElement('button', {className: classes.button})
+
+export default injectSheet(styles)(Button)
+
+    👇
+
+import React from 'react'
+import injectSheet from 'react-jss'
+
+const styles = {
+  button: \\"button-0-0\\"
```

Comment:
Well, I don't know JSS very well, but if you could contribute to this branch to make the transpiled code be something that react-jss could consume (or make it so we could drop the runtime entirely). Also make it so we can handle more code styles. Lots of that will involve just adding more tests and making sure the output will work when run, that'd be great :+1: